libs/damap/src/lib/components/dmp/people/people.module.ts

```diff
@@ -21,6 +23,7 @@ import {MatDialogModule} from '@angular/material/dialog';
     // Materials
     MatCardModule,
     MatSelectModule,
+    MatMenuModule,
```

Comment:
Do we still need that import?

Reply:
Actually no. Tried another solution i forgot to take this out. Sorry for this.